services/queue/schemas/v1/taskqueue-response.yml

```diff
@@ -29,8 +29,8 @@ properties:
   lastDateActive:
     title:        "Task queue Last Date Active"
     description: |
-      Date of the last time this task queue was seen active. `lastDateActive` is updated every 6 hours
-      but may be off by up-to 6 hours. Nonetheless, `lastDateActive` is a good indicator
+      Date of the last time this task queue was seen active. `lastDateActive` is updated every half hour
+      but may be off by up-to half an hour. Nonetheless, `lastDateActive` is a good indicator
```

Comment:
This is a problem with the original comment, rather than your changes, but maybe we can fix it at the same time. I'm not sure what the returned date is, but I suspect it is one of the following:

* Most recent time a worker legally called queue.claimWork for this task queue
* Most recent time a worker legally called queue.claimWork or queue.reclaimTask or queue.resolveTask or queue.createArtifact / .....
* Most recent time a worker legally called queue.XXXXXXX or a worker legally called workerManager.registerWorker / workerManager.XXXXX

In other words, I think the description makes it hard to guess which date it really returns, which means when there is a problem, it is harder to see where the problem is. With a bit more clarification it should help aid troubleshooting when there are problems.